GameData/RealSolarSystem/Compatibility/CustomAsteroids.cfg

```diff
@@ -807,25 +805,24 @@ AsteroidSets
 			// time to prepare a flyby mission
 			type  = MeanAnomaly
 			epoch = Now
-			min = -2.0
-			max = -0.1
+			min = -0.1
+			max = -0.0069
```

Comment:
Even with the enlarged orbits, I'm surprised that these numbers are smaller rather than larger (the new sungrazers have `a` > 9.3 AU, median 90 AU, compared to long-period comets with `a` > 19 AU, median 120-150 AU). Where do comets typically appear with this config?

Reply:
Currently, I'm experimenting with using eccentric anomaly and converting to mean anomaly, the maximum is one year from perihelion with the largest (or smallest?) possible orbit.